modules/schema/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -11508,29 +11806,34 @@ input WhereProposal {
   title: WhereOptionString
 
   """
-  Matches the proposal class.
-  """
-  class: WhereProposalClass
-
-  """
-  Matches the proposal TAC category.
-  """
-  category: WhereOptionEqTacCategory
-
-  """
-  Matches the Target of Opportunity setting.
-  """
-  toOActivation: WhereEqToOActivation
-
-  """
-  Matches the proposal abstract.
-  """
-  abstract: WhereOptionString
-
-  """
-  Matches proposal partners.
-  """
-  partners: WhereProposalPartners
+  Matches on the proposal reference (if any).
+  """
+  reference: WhereProposalReference
+
+#  """
+#  Matches the proposal class.
+#  """
+#  class: WhereProposalClass
+#
+#  """
+#  Matches the proposal TAC category.
+#  """
+#  category: WhereOptionEqTacCategory
+#
+#  """
+#  Matches the Target of Opportunity setting.
+#  """
+#  toOActivation: WhereEqToOActivation
+#
+#  """
+#  Matches the proposal abstract.
+#  """
+#  abstract: WhereOptionString
+#
+#  """
+#  Matches proposal partners.
+#  """
+#  partners: WhereProposalPartners
```

Comment:
Commented out `Where`s. Do we think we'll implement one or more of these later? Should we also comment out or  remove the `WhereProposalClass`, `WhereEqToActivation`, etc?